types/block.go

```diff
@@ -1013,12 +1013,11 @@ type Data struct {
 
 	Evidence EvidenceData `json:"evidence"`
 
-	// The messages included in this block.
-	// TODO: how do messages end up here? (abci) app <-> ll-core?
-	// A simple approach could be: include them in the Tx above and
-	// have a mechanism to split them out somehow? Probably better to include
-	// them only when necessary (before proposing the block) as messages do not
-	// really need to be processed by tendermint
+	// The messages included in this block. TODO: how do messages end up here?
+	// (abci) app <-> ll-core? A simple approach could be: include them in the
+	// Tx above and have a mechanism to split them out somehow? Probably better
+	// to include them only when necessary (before proposing the block) as blobs
+	// do not really need to be processed by tendermint
```

Comment:
Unclear if intentional but this comment still describes messages.

```suggestion
	// The blobs included in this block. 
```

Reply:
oh shoot, whoops

Reply:
fixing in https://github.com/celestiaorg/celestia-core/pull/886